tests/components/tedee/test_config_flow.py

```diff
@@ -100,3 +100,82 @@ async def test_config_flow_errors(
     assert result2["type"] == FlowResultType.FORM
     assert result2["errors"] == error
     assert len(mock_tedee.get_local_bridge.mock_calls) == 1
+
+
+async def test_reauth_flow(
+    hass: HomeAssistant, mock_config_entry: MockConfigEntry, mock_tedee: MagicMock
+) -> None:
+    """Test that the reauth flow works."""
+
+    mock_config_entry.add_to_hass(hass)
+
+    reauth_result = await hass.config_entries.flow.async_init(
+        DOMAIN,
+        context={
+            "source": SOURCE_REAUTH,
+            "unique_id": mock_config_entry.unique_id,
+            "entry_id": mock_config_entry.entry_id,
+        },
+        data={
+            CONF_LOCAL_ACCESS_TOKEN: LOCAL_ACCESS_TOKEN,
+            CONF_HOST: "192.168.1.42",
+        },
+    )
+
+    result = await hass.config_entries.flow.async_configure(
+        reauth_result["flow_id"],
+        {
+            CONF_LOCAL_ACCESS_TOKEN: LOCAL_ACCESS_TOKEN,
+        },
+    )
+    assert result["type"] == FlowResultType.ABORT
+    await hass.async_block_till_done()
```

Comment:
This shouldn't be needed here. (as in, nothing happens in between these 2 asserts. If the test is flakey, it should go after either the async_init or the async_configure